### PR TITLE
Support normalized metric names

### LINKED ISSUE #4547: [Bug]: spanmetrics via connector not working with otel-contrib v0.80.0
### What happened?

The `duration` metric name is no longer normalized with the unit type by default.
The feature gate has been reverted back to off by default, see more details here: https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.80.0

### Steps to reproduce

When trying to run the docker-compose monitor example, no data is visible in Jaeger's Monitor tab.

### Expected behavior

I expect Jaeger to support both normalized and non-normalized metrics names, or at least document the need to have the feature gate enabled.

### Relevant log output

_No response_

### Screenshot

_No response_

### Additional context

_No response_

### Jaeger backend version

v1.46.0

### SDK

OpenTelemetry Collector v0.80.0

### Pipeline

The default provided in the docker-compose monitor example.

### Stogage backend

_No response_

### Operating system

_No response_

### Deployment model

_No response_

### Deployment configs

_No response_

### PR DESCRIPTION
**Pull request by albertteoh**
## Which problem is this PR solving?
- Resolves #4547

## Short description of the changes
- Adds the following boolean parameters to declaratively determine whether the metric name should be modified to match normalization rules as defined [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/translator/prometheus/README.md):
  - `prometheus.query.normalize-calls`
  - `prometheus.query.normalize-duration`
- Separate flags intentional to allow support for backwards compatibility with older OpenTelemetry Collector versions. A single flag is insufficient.
- Motivated by a breaking change in: https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.80.0.

### CODE REVIEW

File: docker-compose/monitor/Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ run-dev: _run-connector
 
 # _run-connector is the base target to bring up the system required for SPM using the new OTEL spanmetrics connector.
 .PHONY: _run-connector
-_run-connector: export OTEL_IMAGE_TAG = latest
+_run-connector: export OTEL_IMAGE_TAG = 0.80.0
```
Comment (albertteoh) [inline docker-compose/monitor/Makefile:21]: Thought to fix the version for a more reliable user experience.

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
DCO?

**Comment by albertteoh** [comment]
> DCO?

Ah, I must have missed signing one commit. Should be good now, thanks.

**Comment by Multiply** [comment]
Thanks for a fast fix. I'll be trying it out right away. 🚀

**Pull request merged**